packages/dialtone-vue3/components/rich_text_editor/extensions/channels/ChannelSuggestion.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <dt-stack
+    direction="row"
+    gap="400"
+  >
+    <dt-icon-hash
+      v-if="!item.locked"
+      size="300"
+    />
+    <dt-icon-lock
+      v-else
+      size="300"
+    />
+    <span>{{ name }}</span>
+  </dt-stack>
+</template>
+
+<script>
+import { DtStack } from '@/components/stack';
+import DtIconHash from '@dialpad/dialtone-icons/vue2/hash';
```

Comment:
There are some errors on the deploy of vue 3 due to this vue 2 references. Vue 2 version looks ok